text/0113-canonical-metadata.md

```diff
@@ -0,0 +1,223 @@
+---
+title: Canonical specification of metadata for LWC modules
+status: DRAFT
+created_at: 2020-05-07
+updated_at: 2020-05-07
+rfc: 
+champion: Aliaksandr Papko (@apapko) | Ravi Jayaramappa (@ravijayaramappa)
+implementation:
+---
+
+# Canonical specification of metadata for LWC modules
+
+Table of Content:
+* [Summary](#summary)
+* [Goals](#goals)
+* [Use Cases](#use-cases)
+* [Current State of Art](#current-state-of-art)
+* [Detailed design](#detailed-design)
+* [References](#reference)
+
+# Summary
+> The [dictionary definition](https://www.merriam-webster.com/dictionary/canonical%20form) of **Canonical form** is `the simplest form of something`. 
+
+This RFC aims to document the metadata collected by statically analyzing the source code of Lightning Web Component(LWC) modules. It documents the standardized shape of metadata.
+
+# Goals
+
+- Completeness
+- Eliminate redundancy: There should be only one way to get a desired information about a module.
+- Consitency in property names
+- Flexibility: Allow for new data to be collected in the future.
+
+# Use Cases
+## Referential Integrity
+One of the key features of the Salesforce platform is referential integrity capabilities. Referential integrity is how the platform allows a given resource to be safely updated and deleted without adverse cascading effects.
```

Comment:
```suggestion
Referential integrity is how the platform allows a given resource to be safely updated and deleted without adverse cascading effects.
```

That first sentence is just fluff and doesn't really support this RFC. I would also provide an example of what referential integrity features this RFC aims to help with - for example renaming components, code, apex etc